doc/userguide/install.rst

```diff
@@ -77,38 +77,96 @@ Rust support::
 
   rustc, cargo
 
-  Not every distro provides Rust packages yet. Rust can also be installed
-  directly from the Rust project itself::
+  Rust packages can be found in package managers but some distros
+  don't provide or provide outdated Rust packages.
+  In case of insufficient version you can install Rust directly
+  from the Rust project itself::
 
-  https://www.rust-lang.org/en-US/install.html
+    1) Install Rust https://www.rust-lang.org/en-US/install.html
+    2) Install cbindgen - if the cbindgen is not found in the repository
+       or the cbindgen version is lower than required, it can be
+       alternatively installed as: cargo install --force cbindgen
+    3) Make sure the cargo path is within your PATH environment
+        e.g. echo 'export PATH=”${PATH}:~/.cargo/bin”' >> ~/.bashrc
+        e.g. export PATH="${PATH}:/root/.cargo/bin"
 
 Ubuntu/Debian
 """""""""""""
 
 Minimal::
 
-    apt-get install libpcre3 libpcre3-dbg libpcre3-dev build-essential libpcap-dev   \
-                    libyaml-0-2 libyaml-dev pkg-config zlib1g zlib1g-dev \
-                    make libmagic-dev libjansson libjansson-dev
+    # Installed Rust and cargo as indicated above
+    apt-get install build-essential git libjansson-dev libpcap-dev \
+                    libpcre3-dev libtool libyaml-dev make pkg-config zlib1g-dev
+    # On most distros installing cbindgen with package manager should be enough
+    apt-get install cbindgen # alternative: cargo install --force cbindgen
```

Comment:
We already covered this in dependencies section above. Running that and then this may cause two diff versions of cbindgen/rust/cargo be installed on the system if I were to just follow along using copy/paste. Thoughts?

Reply:
Rust and cargo are not installed in this step from the repository, only the cbindgen. 

This rust thing gets tricky. I feel like it is more convenient to use the package manager to install rust and cargo directly but I was following recommended practices of the Rust page and that mentions installing Rust through rustup. 

Then in this case again, I feel like it is more convenient to install cbindgen through the package manager. However, in this case, some distros do not contain cbindgen. So as an alternative I listed `cargo install ...` option.

Reply:
I def got befuddled checking rust and cargo but yeah my comment stays for cbindgen. Since we already introduce earlier in the "Dependencies" section to install it cargo way and here pkg mgr way, we can end up with two diff versions of cbindgen if one is following through step by step. Lmk what you think

Reply:
As this is user docs, and this is specifically an Ubuntu an example, we can probably drop mention of `cargo install` for cbindgen, use the packages where available for cbindgen, cargo, rustc. Not cbindgen is only required if building from git, but you don't want to introduce too many *if's* or *ors* into the documentation.